docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Moderated Sessions
+description: Moderated Sessions RBAC Reference
+h1: Moderated Sessions
+---
+
+# Moderated Sessions
+
+Moderated Sessions makes use of RBAC to control access to and within sessions.
+Here is a collection of details on configuring Moderated Sessions.
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who are required to be present to start one.
+
+The system is based around **require policies** and allow policies. Require policies define a set of conditions
+that must be a met for a session to start/run.
+
+**Allow policies** are used to define what sessions a user can join
+and under what conditions they may join a session.
+
```

Comment:
Since I've suggested using the current "RBAC" section intro as the intro for the whole page, let's add a new H2 heading here. Maybe "Configuring Moderated Sessions"? This will include the H3 headings re: specific config options.

```suggestion

## Configuring Moderated Sessions
```